digdag-standards/src/main/java/io/digdag/standards/operator/td/TDResultExportOperatorFactory.java

```diff
@@ -0,0 +1,65 @@
+package io.digdag.standards.operator.td;
+
+import com.google.inject.Inject;
+import com.treasuredata.client.model.TDExportResultJobRequest;
+import io.digdag.client.config.Config;
+import io.digdag.core.Environment;
+import io.digdag.spi.Operator;
+import io.digdag.spi.OperatorContext;
+import io.digdag.spi.OperatorFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+public class TDResultExportOperatorFactory
+        implements OperatorFactory
+{
+    private final Map<String, String> env;
+    private final Config systemConfig;
+    private static Logger logger = LoggerFactory.getLogger(TdTableExportOperatorFactory.class);
+
+    @Inject
+    public TDResultExportOperatorFactory(@Environment Map<String, String> env, Config systemConfig)
+    {
+        this.env = env;
+        this.systemConfig = systemConfig;
+    }
+
+    @Override
+    public String getType() { return "td_result_export"; }
+
+    @Override
+    public Operator newOperator(OperatorContext context) { return new TDResultExportOperator(context); }
+
+    private class TDResultExportOperator
+            extends BaseTdJobOperator
+    {
+        private final String jobId;
+        private final String resultUrl;
```

Comment:
A parameter name `resultUrl` is also used in [TdOperatorFactory](https://github.com/treasure-data/digdag/blob/c67fcb263094fec45f12b054434e08cce985c97a/digdag-standards/src/main/java/io/digdag/standards/operator/td/TdOperatorFactory.java#L121) while TDClient uses `resultOutput` for the same purpose. I choosed `resultUrl` to keep consistency in digdag.